openapi/kas-fleet-manager-private.yaml

```diff
@@ -249,6 +249,16 @@ components:
                       type: string
                     bf2.org/placementId:
                       type: string
+                labels:
+                  type: object
+                  required:
+                    - bf2.org/kafkaInstanceProfileType
+                    - bf2.org/kafkaInstanceProfileQuotaConsumed
+                  properties:
+                    bf2.org/kafkaInstanceProfileType:
+                      type: string
+                    bf2.org/kafkaInstanceProfileQuotaConsumed:
+                      type: string 
```

Comment:
Why labels as opposed to exposing this via annotation: https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/9e943b58907f8f962a8a632d1be8da24cb91a566/openapi/kas-fleet-manager-private.yaml#L242 which we already have?
@VanillaSpoon @JameelB @pb82

Reply:
I believe it needs to be a label because we need to be able to use selectors. It was a suggestion from Fleetshard Services team.